tsconfig.json

```diff
@@ -20,5 +20,5 @@
     "skipLibCheck": true,
     "isolatedModules": true
   },
-  "exclude": ["__tests__", "node_modules/**/*", "src/playground.ts"],
+  "exclude": ["__tests__", "node_modules/**/*", "src/playground.ts", "**/*.d.ts"],
```

Comment:
Everything else looks great, not sure why you made this change though?

Reply:
If you would run build step another time after the initial one, it would produce these errors from `yarn run build`. Think excluding lib dir would be probably better here.
```
yarn run v1.22.5
$ tsc
error TS5055: Cannot write file '/home/ncla/Dev/HLTV/lib/config.d.ts' because it would overwrite input file.

error TS5055: Cannot write file '/home/ncla/Dev/HLTV/lib/endpoints/connectToScorebot.d.ts' because it would overwrite input file.

error
TS5055: Cannot write file '/home/ncla/Dev/HLTV/lib/endpoints/getEvent.d.ts' because it would overwrite input file.

```

Reply:
I reverted this change completely